eessi/testsuite/hooks.py

```diff
@@ -56,6 +60,8 @@ def assign_one_task_per_compute_unit(test: rfm.RegressionTest, compute_unit: str
         # no default set yet, so setting one
         test.default_num_cpus_per_node = int(test.max_avail_cpus_per_node / test.node_part)
 
+    printer.debug(f'assign_one_task_per_compute_unit: default_num_cpus_per_node set to {test.default_num_cpus_per_node}')
```

Comment:
Hmm, wouldn't it be possible to have a `log` function that wraps around `printer.debug`, and automagically determine the function name, so it doesn't have to be manually hardcoded everywhere?

See what we do in the bot at https://github.com/EESSI/eessi-bot-software-layer/blob/5c6b2d0f50ad90aea9f0ebedc8132c16acfc10ec/eessi_bot_event_handler.py#L62

Reply:
good idea, done in https://github.com/EESSI/test-suite/pull/49/commits/54885740c1c5f1b297df1e8af1110f5c52a54941